client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -292,5 +292,30 @@
                 </div>
             </div>
         </div>
+        <div class="col-xm-12 col-md-4 mb-5">
```

Comment:
`col-xm-12` is a mistake I think, and if you want to use `col-xs-12`, you should remove it, it's the default behaviour :-/

Reply:
This is remove. Thanks :)